build/charts/antrea/values.yaml

```diff
@@ -28,8 +28,14 @@ transportInterface: ""
 # -- Network CIDRs of the interface on Node which is used for tunneling or
 # routing the traffic across Nodes.
 transportInterfaceCIDRs: []
-# -- Names of the interfaces on Nodes that are used to forward multicast traffic.
-multicastInterfaces: []
+
+multicast:
+  # -- Names of the interfaces on Nodes that are used to forward multicast traffic.
+  multicastInterfaces: []
+  # -- The interval for antrea-agent to send IGMP query to pods, time uint is second.
+  # Defaults to 125 seconds.
+  igmpQueryInterval: 125
```

Comment:
@tnqn do you think that for time durations we should consistently use go duration strings (e.g. "125s", "2m", "1m30s", ...)?
I think that K8s components do that and we also do that for other Antrea configuration parameters (for the Flow Aggregator IIRC).

Reply:
done